Makefile

```diff
@@ -29,18 +29,10 @@ generate:
 test:
 	go test -count=1 -cover -race -timeout 30s -short `go list ./... | grep -v mocks`
 
-.PHONY: build-base-image
-build-base-image: download-csm-common
-	$(eval include csm-common.mk)
-	@echo "Building base image from $(DEFAULT_BASEIMAGE) and loading dependencies..."
-	./scripts/build_ubi_micro.sh $(DEFAULT_BASEIMAGE)
-	@echo "Base image build: SUCCESS"
-	$(eval BASEIMAGE=mpsc-ubimicro:latest)
-
-# Pre-requisites: RHEL, buildah, podman
 .PHONY: podman
-podman: build-base-image
-	podman build $(NOCACHE) -t csm-metrics-powerscale -f Dockerfile --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE) .
+podman: download-csm-common
+	$(eval include csm-common.mk)
+	podman build $(NOCACHE) -t csm-metrics-powerscale -f Dockerfile --build-arg BASEIMAGE=$(CSM_BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE) .
```

Comment:
Can we also add the `--pull` option to ensure that the most recent `DEFAULT_GOIMAGE` is pulled and used? If I already have a copy of the CSM `nightly` base image, this will not pull the most recent and may result in inconsistent builds.

Reply:
The TechOps pipeline cleans out all local images and pushes a new image as part of the nightly image. So new official images will be built with the latest base image. Some Makefiles have a no-cache version of the podman/docker build. I would prefer to let the developers deal with locally cache images...for now. @Deepak-Ghivari 

There is a later work where we will be standardizing all of the Makefiles across repositories and the local caching can be handled then.